### PR TITLE
added file share list unit test, fixed #84

### LINKED ISSUE #84: Each row of List View has some extra space
If you hover over the list item in the List view. The color changes to light gray(as you can see in a picture below) but below that there is some extra white space, which I think is not required.

screenshot from 2017-01-05 14-48-05

In code there is list-group class with bottom margin 20px, which can be set to 0px and the white space can be removed.

**Problem:**
.list-group {
    margin-bottom: 20px;
    padding-left: 0;
}

**Fix:**
.list-group {
    margin-bottom: 0;
    padding-left: 0;
}

### PR DESCRIPTION
**Pull request by gnehapk**

### CODE REVIEW

**Comment by gnehapk** [comment]
@kamleshverma1 @cloudbehl Please review.

File: src/modules/file-share/file-share-list/file-share-list.html [hunk 1]
```diff
@@ -79,14 +79,14 @@ <h5 ng-if="fileShareCntrl.fileShareList.length !== 0">{{fileShareCntrl.filteredF
...
             <div class="list-group-item">
-                <div class="list-view-pf-left col-md-1">
+                <!-- <div class="list-view-pf-left col-md-1">
                   <i class="fa fa-angle-right" aria-hidden="true"></i>
-                </div>
+                </div> -->
                 <div class="list-view-pf-left col-md-1">
                     <span title="{{fileShare.status}}" ng-if="fileShare.status !=='Stopped'" class="pficon pficon-ok list-view-pf-icon-success"></span>
-                    <span title="{{fileShare.status}}" ng-if="fileShare.status ==='Stopped'" class="fa fa-arrow-circle-o-down"></span>
+                    <span title="{{fileShare.status}}" ng-if="fileShare.status ==='Stopped'" class="fa fa-arrow-circle-o-down list-view-icon-failure"></span>
```
Comment (kamleshverma1) [inline src/modules/file-share/file-share-list/file-share-list.html]: why this css is required - "list-view-icon-failure" . i think "fa fa-arrow-circle-o-down"  will take care for red color icons

File: src/modules/file-share/file-share-list/file-share-list.scss [hunk 2]
```diff
@@ -13,10 +13,10 @@
     }
 
     .list-group {
-        .list-group-item{
-            .list-view-pf-left {
-                text-align: left;
-            }
+        margin-bottom: 0;
+
+        .list-view-icon-failure {
```
Comment (kamleshverma1) [inline src/modules/file-share/file-share-list/file-share-list.scss]: this is not required !

File: src/modules/host/host-list/host-list.html [hunk 3]
```diff
@@ -82,7 +82,7 @@ <h5 ng-if="hostCntrl.hostList.length !== 0">{{hostCntrl.filteredHostList.length}
             <div class="list-group-item">
                 <div class="list-view-pf-left col-md-2">
                     <span title="{{host.status}}" ng-if="host.status ==='UP'" class="pficon pficon-ok list-view-pf-icon-success"></span>
-                    <span title="{{host.status}}" ng-if="host.status !=='UP'" class="fa fa-arrow-circle-o-down"></span>
+                    <span title="{{host.status}}" ng-if="host.status !=='UP'" class="fa fa-arrow-circle-o-down list-view-icon-failure"></span>
```
Comment (kamleshverma1) [inline src/modules/host/host-list/host-list.html]: same as above comment

File: src/modules/host/host-list/host-list.scss [hunk 4]
```diff
@@ -13,11 +13,17 @@
...
     .list-group {
+        margin-bottom: 0px;
+
         .list-group-item{
             .list-view-pf-left {
                 text-align: left;
             }
         }
+
+        .list-view-icon-failure {
```
Comment (kamleshverma1) [inline src/modules/host/host-list/host-list.scss]: same as above comment

File: src/modules/file-share/file-share-list/file-share-list.html (hunk 1 above)
Comment (gnehapk) [inline src/modules/file-share/file-share-list/file-share-list.html]: @kamleshverma1 the color is not red. Hence added this css.

File: src/modules/file-share/file-share-list/file-share-list.scss (hunk 2 above)
Comment (gnehapk) [inline src/modules/file-share/file-share-list/file-share-list.scss]: @kamleshverma1 the color is not red. Hence added this css.

**Pull request merged**